packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -107,7 +111,7 @@ export interface Votes {
 export type HotfixParams = Parameters<Governance['methods']['executeHotfix']>
 export const hotfixToParams = (proposal: Proposal, salt: Buffer): HotfixParams => {
   const p = proposalToParams(proposal, '') // no description URL for hotfixes
-  return [p[0], p[1], p[2], p[3], bufferToString(salt)]
+  return [p[0], p[1], p[2], p[3], bufferToHex(salt)]
```

Comment:
Good catch here! I guess you mean we add the `ensureLeading0x`? I'm inclined to leave it as is, since long-term it'd be good for hex values to be properly prefixed, but if it causes trouble for anyone I'm happy to revert this.